CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Welcome to PASS!
+
+## Project summary
+
+Building out a decentralized application to enable house insecure individuals to securely store their documents electronically. Verified organizations seeking documentation from individuals will utilize PASS to request and review documents from individuals, as well as share documents with individuals. Individuals will need to grant permission to organizations before they can access documents.
+
+## Get started
+
+1. Join [Code for PDX Discord](https://discord.gg/FEX9KUMH). If you run into any issues, shoot an email to our CodeForPDX brigade leader, Hugh: Hugh@codeforpdx.org
+2. Post your GitHub name in the [**#github-access-request**](https://discord.com/channels/1068260532806766733/1078124139983945858) channel and we’ll add you to our [GitHub](https://github.com/codeforpdx/PASS). Even if you don't plan or writing code for the project, please have Hugh get you connected to our GitHub. We keep track of technical and non-technical tasks in GitHub Projects.
+3. Introduce yourself and post your email in the [**#pass**](https://discord.com/channels/1068260532806766733/1075285803137257544) channel so we can add you to our [Google Drive](https://drive.google.com/drive/u/0/folders/1zTEd34K7Eg7rvg71zS6Uzbwrsct2Lx9E?ths=true). You can send that info to Flo privately if you're more comfortable with that. 
+4. Our project is broken into three teams: UX/UI, Project Management, and Developers (back and front end). We are currently meeting in separate groups based on team member availability. The development team meets virtually every Tuesday at 7pm via Google Meet.
+5. Reach out to Flo (Project Manager) on slack to introduce yourself and share how you wish to contribute.
+
+
+## Ready to contribute?
+
+1. Get our github project cloned to your local computer in whatever IDE you prefer. If you are new to Git/GitHub, I have links to learn more as you go. You can also check out [this article](https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github) for a broad overview.
```

Comment:
Instead of "I have links to learn more as you go" on Line 18. It can be changed to, "we have links to learn more as you go"

Reply:
fixed